ddtrace/llmobs/_llmobs.py

```diff
@@ -739,6 +740,7 @@ def submit_evaluation(
         :param tags: A dictionary of string key-value pairs to tag the evaluation metric with.
         :param str ml_app: The name of the ML application
         :param int timestamp_ms: The timestamp in milliseconds when the evaluation metric result was generated.
+        :param dict metadata: A dictionary of additional metadata to include with the evaluation metric.
```

Comment:
what types can the values be?

Reply:
The values can be any arbitrary type which is what makes this different from `tags`. The motivation is that it's useful to attach more complex metadata to evaluation metrics (e.g. a list of parts of the input that were marked as 'harmful'), or the model parameters that were used for an LLM-as-a-judge eval